index.html

```diff
@@ -393,7 +393,7 @@ <h1 class="contact_form_heading">
         </p>
```

Comment:
Please configure linter for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link.